src/magic.c

```diff
@@ -1074,8 +1074,8 @@ TbResult magic_use_power_hold_audience(PlayerNumber plyr_idx, unsigned long mod_
 
 TbResult magic_use_power_chicken(PlayerNumber plyr_idx, struct Thing *thing, MapSubtlCoord stl_x, MapSubtlCoord stl_y, long splevel, unsigned long mod_flags)
 {
-    // If this spell is already casted at that creature, do nothing
-    if (thing_affected_by_spell(thing, SplK_Chicken)) {
+    // If this spell is already casted at that creature or if they have cleanse effect active, do nothing
+    if (thing_affected_by_spell(thing, SplK_Chicken) || thing_affected_by_spell(thing, SplK_Cleanse)) {
```

Comment:
should this not go on line 1089? I believe chicken should be successfully cast, just not doing anything, just like when you try to cast it on a creature that is innately immune to chicken.